### ISSUE TITLE
Search field is covering page

### ISSUE THREAD

**Issue by obeattie** [circleci/circleci-docs#2874]
# Description

A picture is worth a thousand words:
<img width="1233" alt="screenshot 2018-11-04 at 6 40 04 pm" src="https://user-images.githubusercontent.com/5988/47969393-a2f83500-e06e-11e8-9290-ae6cf0ab6e5a.png">

I'm running Google Chrome version 70.0.3538.77 (Official Build) (64-bit)

# Location
Describe where you caught the bug.
If the bug occurs in a specific document,
add the link here.

Basically all the pages on the documentation website.

# Steps to Reproduce
Describe the steps you took to produce the bug.

Open the page

**Comment by obeattie** [comment]
I can no longer reproduce this 🤔
